cms/djangoapps/contentstore/views/tests/test_organizations.py

```diff
@@ -32,3 +41,62 @@ def test_organization_list(self):
         self.assertEqual(response.status_code, 200)
         org_names = json.loads(response.content)
         self.assertEqual(org_names, self.org_short_names)
+
+
+@patch.dict('django.conf.settings.FEATURES', {'ORGANIZATIONS_APP': True})
+@override_switch(settings.ENABLE_EDLY_ORGANIZATIONS_SWITCH, active=True)
```

Comment:
@burhandodhy we should move all these tests to `openedx/features/edly/utils.py`

Reply:
Done